<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallTraceViewer</class>
 <widget class="QWidget" name="CallTraceViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call Trace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="startTraceButton">
       <property name="toolTip">
        <string>Press to start tracing calls and returns</string>
       </property>
       <property name="text">
        <string notr="true">Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stopTraceButton">
       <property name="toolTip">
        <string>Press to stop tracing calls and returns</string>
       </property>
       <property name="text">
        <string notr="true">Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="resizeButton">
       <property name="toolTip">
        <string>Press to resize the columns to their contents</string>
       </property>
       <property name="text">
        <string notr="true">Resize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="toolTip">
        <string>Press to clear the call trace</string>
       </property>
       <property name="text">
        <string notr="true">Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="toolTip">
        <string>Press to save the call trace as a text file</string>
       </property>
       <property name="text">
        <string notr="true">Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="callTrace">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>From</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>To</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startTraceButton</tabstop>
  <tabstop>stopTraceButton</tabstop>
  <tabstop>resizeButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>callTrace</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
